lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1236,6 +1237,18 @@ void addIntelFPGADecorations(
       E->addDecorate(
           new SPIRVDecorateMergeINTELAttr(E, Name.str(), Direction.str()));
     } break;
+    case DecorationBankBitsINTEL: {
+      SmallVector<StringRef, 1> A;
```

Comment:
I'm not sure 1 is a good choice for this parameter.
Can we have a more meaningful name then just A? Something like `Bits` maybe.